### ISSUE TITLE
[QUERY] Multiple questions about beta version of package Azure.Maps.Routing

### ISSUE THREAD

**Issue by worming004** [Azure/azure-sdk-for-net#44834]
### Library name and version

Azure.Maps.Routing

### Query/Question

Hello,

Currently, nuget is available as beta version. https://www.nuget.org/packages/Azure.Maps.Routing. With version 1.0.0-beta.2

# Question 1

Does beta means there is no garantee to not make breaking change ? Or anything else ?

# Question 2

Does the nuget is Prod ready ? Does beta means we should not use it in production ?

# Question 3

Does there is plan to deliver a stable(non-beta) version of this nuget ?

**Comment by dubiety** [comment]
Hi @worming004,

1. It's not guaranteed to make non-breaking changes between beta versions. Based on our release, we follow the semantic versioning. Only major version update will have breaking changes. E.g. `v1.0.0-beta1` to `v2.0.0-beta1`
2. The SDK are prod ready and we don't expect any major changes. However, once we have GA version SDK, there may be some changes between beta and GA SDK. Like parameter/function name changes. The potential changes are still unknown and we're still working with SDK core team.
3. We still in beta versions only because we'll release GA version after we finish all the API categories. We're working hard to ship all the packages and make them GA, but there's no explicit ETA right now.

In short, we will GA our SDK, but still need some time. If you're worried about the changes, please wait until the GA version, but in the meantime, I don't expect major changes between beta versions (unless it's major version change). Let me know if you need any further clarification and thank you for your patience. I'll keep this issue open and will inform you once the SDKs are GA.
